infra/aws/README.md

```diff
@@ -0,0 +1,83 @@
+Building a cluster on AWS
+=========================
+
```

Comment:
Perhaps start with a requirements list:

"""
For following these instructions, you will need:

- packer installed
- terraform installed
- access to the Test Ground 1Password Vault
- An account in the AWS Test Ground account

...
"""